storage/gcloud/aio/storage/bucket.py

```diff
@@ -47,10 +47,20 @@ def __init__(self, storage, name: str) -> None:
     async def list_blobs(self, prefix: str = '',
                          session: Optional[Session] = None) -> List[str]:
         params = {'prefix': prefix}
-        content = await self.storage.list_objects(self.name, params=params,
+        content = await self.storage.list_objects(self.name,
+                                                  params=params,
                                                   session=session)
-
+        next_content = {}
+        next_content['nextPageToken'] = content.get('nextPageToken')
+        while next_content.get('nextPageToken'):
+            params['pageToken'] = next_content['nextPageToken']
+            next_content = await self.storage.list_objects(self.name,
+                                                           params=params,
+                                                           session=session)
+            content['items'] = content['items']\
```

Comment:
General pro-tip here: `lists` support `append` and `extend` methods, which are generally preferred as being a bit more readable then using the arithmetic operators. In this case, what you've written here could instead be formulated as:
```python
content['items'].extend(next_content.get('items', list()))
```